GoogleForCreators/web-stories-wp
# Fixed rtl placement for popups

## Context

This fixes the popup placement calculation in RTL contexts by flipping some left and right logic in those cases.

As I fixed this, I noticed a bug in the color picker itself (which was never visible before as the color picker itself was never usable in RTL) with the pointer being wrongly aligned - I fixed this too.

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2021-05-04 at 16 56 51](https://user-images.githubusercontent.com/637548/117068955-e4c6cc80-acf9-11eb-965b-e9d6f13e559b.png) | ![Screen Shot 2021-05-04 at 16 54 25](https://user-images.githubusercontent.com/637548/117068959-e5f7f980-acf9-11eb-8c26-8298c94bf560.png) |
| ![Screen Shot 2021-05-04 at 17 08 26](https://user-images.githubusercontent.com/637548/117070245-797dfa00-acfb-11eb-96af-8687e9795ccb.png) | ![Screen Shot 2021-05-04 at 17 07 34](https://user-images.githubusercontent.com/637548/117070256-7aaf2700-acfb-11eb-8c96-98f309069eac.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Flip WP admin to RTL
2. Select the background element
3. Open the color picker
4. Observe that it is correctly placed adjacent to but not overlapping the design panel
5. Use the color picker saturation selector to pick a color
6. Observe that the sample circle is correctly placed relative to the selection

### UAT




## Checklist




---

Fixes #6804
